src/test/java/com/e451/service/CampaignDescriptionServiceTest.java

```diff
@@ -106,10 +106,10 @@ public void testCampaignDescriptionServiceCampaignDescriptionRecordCampaignDescr
                 , CampaignDescription.getDescription());
         Assert.assertEquals("The Start_Day was incorrectly converted"
                 , CAMPAIGN_DESCRIPTION_START_DAY1
-                , CampaignDescription.getStart_day());
+                , CampaignDescription.getStartDay());
```

Comment:
I know it makes things easier to compile when you comment things out, but I really, _really_ hate commas at the beginning of a line of code.